src/loan-estimate/loan-estimate-1.html

```diff
@@ -11,7 +11,8 @@
         "left": "16.17%",
         "top": "15.02%",
         "width": "21.56%",
-        "height": "4.50%"
+        "height": "4.50%",
+        "new": true
```

Comment:
What if we called this property something like 'descriptionTag' and made 'new' its value? That would make it possible for us to apply different tags in future, if we ever wanted to do so...